portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,30 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random quote to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomQuote() {
+  const quotes =
+      ["If you're the CEO of a company, and you're dumb enough to leave your log-in info on a Post-It note on your desk, while the people that you ripped off are physically in your office, it's not a hack. It's barely social engineering. It’s more like natural selection." ,
```

Comment:
It is in general good practice to move text on next line if character count exceeds 100. This helps in code readability.